Directory.Build.targets

```diff
@@ -20,4 +20,11 @@
     <NETCoreAppMaximumVersion>$(MajorVersion).$(MinorVersion)</NETCoreAppMaximumVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
```

Comment:
><PropertyGroup> [](start = 2, length = 15)

It'd be good to add a comment that this is a workaround for a bug in the SDK and link to the issue that fixes it.

Reply:
But the workaround is adding the file to EmbeddedFiles right?

Reply:
Yes, any AssemblyAttributes files that got generated out of order before this flows through https://github.com/dotnet/sdk/pull/10613